src/backend/distributed/commands/function.c

```diff
@@ -1305,6 +1305,10 @@ CreateFunctionStmtObjectAddress(Node *node, bool missing_ok)
 	FunctionParameter *funcParam = NULL;
 	foreach_ptr(funcParam, stmt->parameters)
 	{
+		if (funcParam->mode == FUNC_PARAM_OUT || funcParam->mode == FUNC_PARAM_TABLE)
```

Comment:
With this change we will now search functions with different arguments than before. It's unclear to me if we will still find the function that we are looking for. Can you add a test (or point to an existing test) shows that this doesn't break stuff.